vars/buildPlugin.groovy

```diff
@@ -28,17 +34,33 @@ def call(Map params = [:])
 		'master': 'leia',
 		'Leia': 'leia'
 	]
+	def PPAS_VALID = [
+		'nightly': 'ppa:team-xbmc/xbmc-nightly',
+		'unstable': 'ppa:team-xbmc/unstable',
+		'stable': 'ppa:team-xbmc/ppa',
+		'wsnipex-test': 'ppa:wsnipex/xbmc-addons-test'
+	]
+	def PPA_VERSION_MAP = [
+		'master': 'nightly',
+		'Leia': 'unstable'
```

Comment:
Shouldn't both (actually `master` and `Leia` is the same, as not branched yet) currently get uploaded to `nightly` and `unstable`?

Reply:
as long as master and leia is the same, we don't want to cause double load for launchpad. Binaries can be copied to unstable on launchpad

Reply:
is that automated?

Reply:
no

Reply:
Is there a way to automate it?

Reply:
maybe via the API. But once leia is branched it will upload to unstable(we'll have to change it to stable after v18 release)

Reply:
after branching, same situation only ppas change
Leia to stable and unstable

Reply:
not really. Once leia is final, all updates from leia branch will go directly to the stable ppa. Unstable won't be used anymore until v19 beta

Reply:
RC versions for minor releases will go to stable?

Reply:
since when do we addons have RCs?
I'm not strictly against uploading to multiple PPAs, so I have added the capability to do so. But I it makes sense to do it automatically.

Reply:
I was talking about Kodi versions. Where have you published 17.1rc1-Krypton and 17.4rc1-Krypton?